types/recharts/index.d.ts

```diff
@@ -178,7 +178,9 @@ export class CartesianGrid extends React.Component<CartesianGridProps> { }
 
 export interface CellProps {
 	fill?: string;
+	fillOpacity?: number;
```

Comment:
Since this is part of `CSSStyleDeclaration` I would suggest to use directly `Partial<CSSStyleDeclaration>`